src/Pages/MyPage/MyPageComponent/ClassCard.js

```diff
@@ -0,0 +1,173 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { FaRegHeart, FaHeart, FaThumbsUp, FaFireAlt } from "react-icons/fa";
+import styled from "styled-components";
+const ClassCard = ({
+  cardWidth,
+  imgSrc,
+  isOpening,
+  category,
+  mentor,
+  description,
+  likeCount,
+  cheeredRate,
+  thumbsUp,
+  originPrice,
+  discount,
+  month,
+}) => {
+  const [isLiked, setIsLiked] = useState(false);
+  const [pricePerMonth, setPricePerMonth] = useState(0);
+  useEffect(() => {
+    if (month && originPrice && discount) {
+      const discountedPrice = originPrice * discount;
+      setPricePerMonth(discountedPrice / month);
+    }
+  }, [discount, originPrice, month]);
+  return (
+    <ClassCardComponent cardWidth={cardWidth}>
+      <Link>
+        <HoverImgBox>
+          <ImgWithProps
+            src={
+              imgSrc ||
+              "https://images.unsplash.com/photo-1589395595558-18e333cf6341?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
+            }
+            alt="ClassThumbnails"
+          />
+          <LikeBtn onClick={() => setIsLiked(!isLiked)}>
+            {isLiked ? <FaHeart fill="#d42839" /> : <FaRegHeart fill="#f8f8f8" />}
+          </LikeBtn>
+        </HoverImgBox>
+        <TextDiv>
+          <Bold>
+            {category}, {mentor}
+          </Bold>
+          <p>{description}</p>
+          <Rates>
+            {isOpening && (
+              <span className="red">
+                <FaFireAlt /> {cheeredRate}% 달성
+              </span>
+            )}
+            <span>
+              <FaHeart /> {likeCount}
+            </span>
+            {!isOpening && (
+              <span>
+                <FaThumbsUp /> {thumbsUp}%
+              </span>
+            )}
+          </Rates>
+        </TextDiv>
+        <TextDiv className="noBorder">
+          {isOpening ? (
+            <Button>응원하기</Button>
+          ) : (
+            <>
+              <p>
+                <CentereLinedSpan>{originPrice}원</CentereLinedSpan>
+                <span className="red">{discount * 100}%</span>
+              </p>
+              <p>
+                <Bold className="cost">월 {pricePerMonth}원</Bold>
+                <span>({month}개월)</span>
+              </p>
+            </>
+          )}
+        </TextDiv>
+      </Link>
+    </ClassCardComponent>
+  );
+};
+const ClassCardComponent = styled.div`
+  width: ${(props) => props.cardWidth || 267}px;
+  color: #1b1c1d;
+`;
+const HoverImgBox = styled.div`
```

Comment:
```suggestion
};

const ClassCardComponent = styled.div`
  width: ${({ cardWidth }) => cardWidth || 267}px;
  color: #1b1c1d;
`;

const HoverImgBox = styled.div`
```
- 마찬가지로 한줄씩 띄워주세요
- 여기서도 구조분해 할당 가능합니다

Reply:
ClassCard.js는 팀원이 작업했던 컴포넌트가 아직 merge가 안됐어서 잘 붙여지는지 보려고 일단 따로 파일 받아서 임포트했습니다! 현재는 merge되어서 저 파일은 지우고 머지된 파일로 임포트 경로 수정했습니다!